### DISCUSSION TITLE
keygen-cli and self hosted solution

### DISCUSSION THREAD

**Discussion by fbergamelli** [keygen-sh/community#84] [General]
Hello,

is it possible to have your keygen-cli work with a self hosted instance of keygen.sh ? I do not really know go at all and a simple "grep" seems to not bring up the server entry.

thanks

Federico

**Comment by ezekg** [comment]
Yes. This was actually introduced in the latest version of the CLI via the `--host` flag.

```bash
keygen new --version 1.0.0 --host https://licensing.example.com
```

Use `keygen upgrade` to grab the latest version.
